src/modules/navigation/components/Navigation/Navigation.module.scss

```diff
@@ -215,11 +215,19 @@ $c-navigation-decoration: url("data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDo
 		grid-column: 3/4;
 		justify-self: end;
 
-		// Ward: first child element is MaterialRequestCenterButton, which has a blade
-		// the other navigation items pop up through the blade if the following z-index is removed
 		ul {
-			li:first-child {
-				z-index: get-z-layer("filter");
+			grid-gap: 0;
+
+			li {
+				// Ward: first child element is MaterialRequestCenterButton, which has a blade
+				// the other navigation items pop up through the blade if the following z-index is removed
+				&:first-child {
+					z-index: get-z-layer("filter");
+				}
+
+				&:nth-child(2) {
+					padding: 0 1.5rem 0 0.5rem;
```

Comment:
can't we do this using the grid-gap option instead of doing a padding for one of the items?
eg:
```
.c-navigation__list {
    display: grid;
    grid-auto-flow: column;
    grid-gap: 1.5rem;
}
```